pallets/committee/src/lib.rs

```diff
@@ -170,6 +173,9 @@ pub mod pallet {
 		/// A new consituent has been added
 		/// \[constituent_address]
 		NewConstituent(AccountIdFor<T>),
+		/// A member has been removed
+		/// \[member_type, address]
+		RemoveMember(MemberType, AccountIdFor<T>),
```

Comment:
I'd rather switch the order of params here
```suggestion
		MemberRemoved(AccountIdFor<T>, MemberType),
```